assets/scss/pages/index/_category.scss

```
@@ -0,0 +1,49 @@
+.category-row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: -6px;
+  margin-right: -6px;
+}
+
+.category-col-9 {
+  width: calc(100% / 12 * 9);
+  padding-left: 6px;
+  padding-right: 6px;
+}
+
+.category-col-3 {
+  width: calc(100% / 12 * 3);
+  padding-left: 6px;
+  padding-right: 6px;
+}
+
+.category-card-img {
+  height: 350px;
+
+  transition: transform 0.4s ease;
+  transform: scale(1);
+
+  @include media-breakpoint-up(lg) {
+    height: 560px;
+  }
+}
+
+.category-card:hover .category-card-img {
+  transform: scale(1.1);
```

```suggestion
  transform: scale(1.1);
  transition: transform 0.4s ease;
```
如果把23行的transition，改到這邊的話，是不是比較好呢?

---

不行。希望讓 hover 開始和結束都有平滑的過渡效果時，transition 屬性必須寫在元素的「初始狀態」上。
這樣改會在 hover 結束時瞬間變回 1x 大小而有點突兀。